### PR TITLE
Update Laravel QuickStart

### PR DESCRIPTION
**Pull request by joshcanhelp**
The Laravel QuickStart document and seed project have bugs and have
fallen behind in general. This commit and all subsequent in this branch
will update the QS document from start to finish. The final product
will be PRed against the seed project so the two match up.

Seed project update is [now in PR](https://github.com/auth0-samples/auth0-laravel-php-web-app/pull/25) so the repo that this update points to will be ready soon. Since this is such a large update, I wanted to get it into review ASAP. 

Thank you!

https://auth0-docs-content-pr-5894.herokuapp.com/docs/quickstart/webapp/laravel/01-login

### CODE REVIEW

**Comment by joshcanhelp** [comment]
@aaguiarz - Heads-up on this. Not adding you as an official reviewer but you might want to look through it.

**Review by jeffreylees — APPROVED** [review]

**Comment by joshcanhelp** [comment]
@mpaktiti - Adding you for final review ... thank you!

**Review by mpaktiti — APPROVED** [review]
LGTM

**Comment by joshcanhelp** [comment]
Thanks @mpaktiti! Marking this `do-not-merge` until the seed project change goes live. I'll ping here when that's ready.

File: articles/quickstart/webapp/laravel/01-login.md [hunk 1]
```diff
@@ -41,77 +60,154 @@ Optionally, if you want to use the [facade](http://laravel.com/docs/facades) cal
...
 
-In other words, you need to select a uri (for example `/auth0/callback`) and configure it in your [Auth0 admin page](${manage_url}/#/applications) and also, add it as a route in `routes/web.php`:
+To keep sensitive data out of version control and allow for different testing and production instances, we recommend using the `.env` file Laravel uses to load other environment-specific variables:
+
+```
+// .env
+// ...
+AUTH0_DOMAIN=${account.namespace}
+AUTH0_CLIENT_ID=${account.clientId}
+AUTH0_CLIENT_SECRET=${account.clientSecret}
```
Comment (jeffreylees) [inline articles/quickstart/webapp/laravel/01-login.md]: This should just say `'YOUR_CLIENT_SECRET'`, as we're steering away from using this particular variable in docs, and I'd not even thought about that until it just came up in a conversation and I remembered this PR :)

Comment (joshcanhelp) [reply articles/quickstart/webapp/laravel/01-login.md]: @jeffreylees - You're welcome to change it back :)

**Comment by jeffreylees** [comment]
Absolutely, just didn't want it to merge with it. Will update later this
evening

**Comment by joshcanhelp** [comment]
NP. We should have the project ready by Friday at the latest, would be great to merge around the same time if possible.

**Comment by mpaktiti** [comment]
@joshcanhelp Hands up, I'm flying back home tomorrow so if I'm not online to merge ping @khorne3 and she can do it.

**Comment by joshcanhelp** [comment]
@mpaktiti - Actually, the seed update was just made right now so merge away if you're able. Thank you!

**Pull request merged**